.editorconfig

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
Not really sure why this file was added.

Reply:
My bad, it's not related to the topic, but I think it's not a bad idea, it change IDE behaviour to match what is in the file. 

Example, I prefer indent with 2 spaces, with that my Atom automatically change for 4 spaces for this project.

Reply:
That's true. Let's leave it in then!